### DISCUSSION TITLE
Where are angular release notes published to?

### DISCUSSION THREAD

**Discussion by Wallabeng** [vmware/clarity#6611] [Q&A] [answered]
I just can find the code for the angular release notes, but no published site (at least not linked in official documentation).
e.g. this one here https://github.com/vmware/clarity/blob/angular/projects/website/src/releases/13.x/13.0.0.html

**Comment by hippee-lee** [comment, marked as answer]
Hi @Wallabeng - the release notes are [here](https://angular.clarity.design/news).

**Reply by Wallabeng** [reply]
Thanks a lot! Would be nice to have them linked anywhere in the official docs.
